neural/run_workflow.py

```diff
@@ -216,8 +217,7 @@ def run_neural_discern(data_partition, dsettypes, bertmodel, config, options, wr
     docid_attnweights_map = {dsettype: {} for dsettype in data_loaders if dsettype in {'validation', 'test'}}
     # store sentences' attention weights
 
-    if('validation' in data_loaders):
-        m_state_dict_dir = create_directory(os.path.join(wrk_dir, 'model_statedict'))
+    m_state_dict_dir = create_directory(os.path.join(wrk_dir, 'model_statedict'))
```

Comment:
currently you create "model_statedict" directory by default. Typically, you need that when you are training and validating on a validation set. It is not that important here -- that's why there is an `if` statement to check the case of `training and validation`.